test/e2e/serial/README.md

```diff
@@ -16,7 +16,7 @@ run in isolation with any other workload, and in isolation to each other.
 
 IOW, the suite expects to run against a pristine, unloaded cluster to validate it.
 
-It is responsability of the tests in this suite to clean up properly and to restore the cluster state as
+It is responsibility of the tests in this suite to clean up properly and to restore the cluster state as
```

Comment:
this (and following) is a good change and can be split off to its own tiny PR and merged immediately

Reply:
Thanks for raising that. Done here:
https://github.com/openshift-kni/numaresources-operator/pull/433